src/main/java/com/google/sps/servlets/UserServlet.java

```diff
@@ -24,9 +25,10 @@
    */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
     //dummy data for user
-    String username = "Sam";
-    
+    String authenticatedEmail = "sam@gmail.com";
+
     Gson gson = new Gson();
```

Comment:
Consider initialising this as a class variable rather than an instance in the servlet, so the object is only created once rather than on every call, and can serve both doGet and doPost.